include/oneapi/dpl/experimental/kt/esimd_scan.h

```diff
@@ -179,6 +169,17 @@ void inclusive_scan(sycl::queue q, InputIterator first, InputIterator last, Outp
         simd<ValueType,VL> v;
         ValueType carry31;
 
+        // propogate carry in from previous block
+        ValueType carry_in = 0;
+        if (b > 0) {
+        #if 0
+        ci = scalar_load<uint>(result,offset+(b*blockSize-1)*sizeof(uint32_t));
+        #else
```

Comment:
There should be no `#if 0` blocks of code; please either remove or specify a reasonable preprocessor condition. This code will not even compile, because of `ci` being not declared.

Reply:
The line in the `#if 0` is no longer needed, so I have removed it. I will prepare a PR for the other places this has been done when working on the prototype.

Reply:
Its worth noting that this PR is targeting a branch for shared development of a prototype, rather than oneDPL main.